src/project.rs

```diff
@@ -378,12 +379,16 @@ impl Projects {
         let previous_height = self.scroll.max_content_height();
 
         // strange solution to make gtk assertions happy
-        if previous_height < treeview_height {
-            self.scroll.set_max_content_height(treeview_height);
-            self.scroll.set_min_content_height(treeview_height);
-        } else if previous_height > treeview_height {
-            self.scroll.set_min_content_height(treeview_height);
-            self.scroll.set_max_content_height(treeview_height);
+        match previous_height.cmp(&treeview_height) {
+            Less => {
```

Comment:
I'm not sure whether this improves readability. It at least forces to consider all there cases

Reply:
TBH I think I'd prefer these ones be `#[clippy:allow(…)]`d

Reply:
TBH I think I'd prefer this ones be `#[clippy:allow(…)]`d